pkg/gui/tasks_adapter.go

```diff
@@ -98,21 +98,15 @@ func (gui *Gui) getManager(view *gocui.View) *tasks.ViewBufferManager {
 			},
 			func() {
 				// Need to check if the content of the view is well past the origin.
-				// It would be better to use .ViewLinesHeight here (given it considers
```

Comment:
nothing broke when I made this change but I have a suspicion something might actually break. Will play it by ear